### ISSUE TITLE
[Topics] Style Last active topics blocks

### ISSUE THREAD

**Issue by tuxpiper** [project-pheme/project-pheme-platform-client#23]
1. The title could look better
2. Style topic information
3. (journalist request) Would it be possible to lay the topics out in up to 2/3 columns depending on width? (flexible grid?)

![screen shot 2017-01-09 at 10 08 37](https://cloud.githubusercontent.com/assets/467393/21761628/82ee11b0-d654-11e6-82a8-50cddc8e153f.png)

**Comment by tuxpiper** [comment]
Thanks @middle8media ! Would you like to take a look about styling the content of the cells a bit? I've added one more line of contents, it's static now, I'll make it dynamic later on.

<img width="894" alt="screen shot 2017-01-15 at 22 40 58" src="https://cloud.githubusercontent.com/assets/467393/21965576/d837ac66-db73-11e6-94a9-8c9e863b1ca4.png">

**Comment by sethburtonhall** [comment]
Here is where I have landed ...

![image](https://cloud.githubusercontent.com/assets/1033332/21999667/44e4d3bc-dc09-11e6-8bc4-6aa0f410850f.png)

FYI, what you see is the max width (both search and content box), therefore I reverted the grid. I know the client requested it, but I would like to suggest this approach. IMO, the max width takes away the need for the grid, plus it is less troublesome.

Thoughts? I will push and we can change as needed.
